examples/platforms/simulation/system.c

```diff
@@ -67,52 +67,68 @@ void otSysInit(int aArgCount, char *aArgVector[])
 {
     char *   endptr;
     uint32_t speedUpFactor = 1;
-    int      argIndex      = 0;
+
+    struct option long_options[] = {
+        {"sleep-to-tx", no_argument, 0, 's'},
+        {"time-speed-up-factor", required_argument, 0, 't'},
```

Comment:
consistent with posix app.
```suggestion
        {"time-speed", required_argument, 0, 't'},
```